models/marts/entr/entr.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+
+models:
+  - name: fct_entr_plant_data
+    description: generic ENTR fact table containing plant-level information
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - plant_id
+            - date_time
+            - entr_tag_id
```

Comment:
I'm guessing we'll come across needing to add support for multiple sources at some point